flexmeasures/api/v3_0/tests/conftest.py

```diff
@@ -26,7 +34,22 @@ def setup_api_fresh_test_data(
     print("Setting up fresh data for API 3.0 tests on %s" % fresh_db.engine)
     for sensor in Sensor.query.all():
         fresh_db.delete(sensor)
-    add_gas_sensor(fresh_db, setup_roles_users_fresh_db["Test Supplier User"])
+    gas_sensor = add_gas_sensor(
+        fresh_db, setup_roles_users_fresh_db["Test Supplier User"]
+    )
+    return {gas_sensor.name: gas_sensor}
+
+
+@pytest.fixture(scope="function")
```

Comment:
Why not call this function directly in `setup_api_fresh_test_data`? 

Makes more sense to me. Then we don't need to mention it in `test_get_sensor_data`.

Reply:
That would mean adding the gas measurements for every test that uses the `setup_api_fresh_test_data` fixture, while not every test needs it.

Reply:
I believe we often add data for the whole set of tests, for easier reading. It doesn't take long, either.

Reply:
The POST data tests are actually easier to read without having preexisting measurements set up. They assume there is no data before they POST some, and do a simple query for all of the sensor's data after posting.

Reply:
Nevertheless, while resolving merge conflicts I implemented your suggestion in #542.